packages/app-bridge/src/tests/TemplateDummy.ts

```diff
@@ -10,6 +10,7 @@ export class TemplateDummy {
             name: 'A template',
             description: 'A description',
             previewUrl: 'https://preview.url',
+            creationFormUri: `/publishing/template/${String(id)}?referer=test.frontify.com`,
```

Comment:
```suggestion
            creationFormUri: `/publishing/template/${id}?referer=test.frontify.com`,
```

Sorry stupid me, you don't need to explicitly convert it, should work like that